src/main/java/seedu/address/model/client/SuggestionType.java

```diff
@@ -0,0 +1,57 @@
+package seedu.address.model.client;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.AppUtil.checkArgument;
+
+/**
+ * Represents a SuggestionType in the address book.
+ * Guarantees: immutable; suggestion type is valid as declared in {@link #isValidSuggestionType(String)}
+ */
+public class SuggestionType {
```

Comment:
Just a suggestion, should `SuggestionType` be an `Enum` instead? Since it is only taking `frequency`, `available` and `contract` as valid inputs.

Reply:
possible, I initially put it as an enum but it seemed too different from the rest of the code base. What do you think @qwoprocks

Reply:
I think it's fine to leave it as a class for now, since if we change it to an enum it might be hard to integrate. Maybe can just KIV as a TODO.